gf-server-resources/autogenerated/swagger.yml

```diff
@@ -5,7 +5,7 @@ info:
   description: A simple REST API for providing basic CRUD-access to the employees
     in a Golden Fleece database.
 servers:
-  - url: http://localhost:3501
+  - url: undefined:undefined
```

Comment:
Something wrong. If url is undefined will be errors

Reply:
Done